viewer/viewer.js

```diff
@@ -4619,7 +4064,7 @@ app.getpost( // build query endoint (POST or GET) - uses fillQueryFromBody to
 
 app.getpost( // sessions csv endpoint (POST or GET) - uses fillQueryFromBody to
   // fill the query parameters if the client uses POST to support POST and GET
-  ['/api/sessions/csv', /\/sessions.csv.*/],
+  [/\/api\/sessions\/csv*/, /\/sessions.csv.*/],
```

Comment:
This should be .* instead of * for a regex.
Would you be willing to compromise with /\/api\/sessions[/.]csv.*/ then I could call this with both /api/sessions/csv and /api/sessions.csv